rubix-common/src/main/java/com/qubole/rubix/common/metrics/NoOpReporter.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2019. Qubole Inc
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License. See accompanying LICENSE file.
+ */
+package com.qubole.rubix.common.metrics;
+
+import com.codahale.metrics.MetricRegistry;
+import org.apache.hadoop.conf.Configuration;
+
+import java.io.IOException;
+import java.util.Optional;
+
+public class NoOpReporter implements CustomMetricReporter {
+    public NoOpReporter(Configuration configuration, Optional<MetricRegistry> metricRegistry) {
```

Comment:
we are not using the parameters anywhere, so can get rid of them.

Reply:
This constructor is needed in CustomMetricsReporterProvider